idaes/generic_models/unit_models/pressure_changer.py

```diff
@@ -479,34 +524,6 @@ def actual_work(b, t):
                     b.work_isentropic[t] * b.efficiency_isentropic[t]
                 )
 
-        # Add head expressions (often calcuated from performance curve)
-        try:
-            @self.Expression(self.flowsheet().config.time)
-            def head_isentropic(b, t): # units are energy/mass e.g. J/kg
-                if hasattr(b.control_volume.properties_in[t], "flow_mass"):
-                    return (b.work_isentropic[t] /
-                        b.control_volume.properties_in[t].flow_mass)
-                else:
-                    return (b.work_isentropic[t] /
-                        b.control_volume.properties_in[t].flow_mol /
-                        b.control_volume.properties_in[t].mw)
-
-            @self.Expression(self.flowsheet().config.time)
-            def head(b, t): # units are energy/mass e.g. J/kg
-                if hasattr(b.control_volume.properties_in[t], "flow_mass"):
-                    return (b.work_mechanical[t] /
-                        b.control_volume.properties_in[t].flow_mass)
-                else:
-                    return (b.work_mechanical[t] /
-                        b.control_volume.properties_in[t].flow_mol /
-                        b.control_volume.properties_in[t].mw)
-
-        except PropertyNotSupportedError:
-            _log.warning(
-                "flow_mass or flow_mol and mw not supported by property package"
-                " but are required for isentropic pressure changer head"
-                " calculation")
-
         # Add performace curve block.  If this block is empty that's fine
         self.performance_curve = IsentropicPerformanceCurve(
             default=self.config.isentropic_perfomance_curves)
```

Comment:
Typo: perfomance

This appears in a few places, so you might want to do a search and replace.